### ISSUE TITLE
All Pokemon still escaping

### ISSUE THREAD

**Issue by LunarGaze** [mfron/PokemonGo-Bot#92]
All Pokemon still get away even after latest update.

**Comment by Alayden** [comment]
@LunarGaze Can you explain to me how to download the latest file , please?

**Comment by LunarGaze** [comment]
It's in the .zip file already.

**Comment by Alayden** [comment]
A link , please?

**Comment by LunarGaze** [comment]
https://github.com/mfron/PokemonGo-Bot/archive/master.zip

**Comment by Imthatbeatzz** [comment]
Yeah for me too, all pokemons escape and my backpack is full so its not using the pokeballs that are available

**Comment by Imthatbeatzz** [comment]
@mfron

**Comment by Alayden** [comment]
@LunarGaze I really don't get it , where do you find the .exe program to open the bot ?
![image](https://cloud.githubusercontent.com/assets/20782730/17313416/bd89a950-585b-11e6-8648-60758c1a479a.png)

I'm sorry to bother you but it could help me?

**Comment by LunarGaze** [comment]
You need Microsoft Visual Studio 2015.

**Comment by Alayden** [comment]
@LunarGaze Thank you , sir :)

**Comment by mfron** [comment]
The compiled version is not the latest version. Pokemons will escape.
This is patched on the not compiled version. You need to build it for get the version with the patch.

**Comment by LunarGaze** [comment]
I did build it, the Pokemon are still escaping.

**Comment by Imthatbeatzz** [comment]
Thats true, if you could get the fix into the release it would help alot :)

**Comment by mfron** [comment]
Im running 5 bots, and me not, what about you @Imthatbeatzz  ?

**Comment by Alayden** [comment]
@LunarGaze Hey , I come back to you after you said that I have to download Microsoft Visual Studio 2015. 

I did it and now I get this ? 

![image](https://cloud.githubusercontent.com/assets/20782730/17313750/bf93726e-585e-11e6-8358-165779c32634.png)

I could open my bot before I downloaded it ?

**Comment by Alayden** [comment]
@mfron You should remake your downloading files , it's really , really a mess. ( Also , give your paypal , i'll give you a thing for your work. ) 

Look at this other bot 
![image](https://cloud.githubusercontent.com/assets/20782730/17313792/17c0c5cc-585f-11e6-8136-a44335bce320.png)

It's simple , one folder with a .exe. You should try it too.

**Comment by Imthatbeatzz** [comment]
@mfron Can you paste what i should replace in here if i got it wrong please :) Yeah, give us your paypal, and i can give you something not that big but atleast something :)

**Comment by Imthatbeatzz** [comment]
@mfron I know the problem, my bag is full which dont containes any pokeballs, the bot seems to not clean up the bag? why not :(

**Comment by stickmick12** [comment]
How i can open with virsual studio more account in the same time?
